applications/tari_base_node/Cargo.toml

```diff
@@ -38,7 +38,7 @@ futures = { version = "^0.3.16", default-features = false, features = ["alloc"]
 qrcode = { version = "0.12" }
 log = { version = "0.4.8", features = ["std"] }
 log-mdc = "0.1.0"
-log4rs = { version = "1.2.0", default_features = false, features = ["config_parsing", "threshold_filter", "yaml_format", "console_appender", "rolling_file_appender", "compound_policy", "size_trigger", "fixed_window_roller"] }
+log4rs = { git = "https://github.com/brianp/log4rs.git", default_features = false, features = ["config_parsing", "threshold_filter", "yaml_format", "console_appender", "rolling_file_appender", "compound_policy", "size_trigger", "fixed_window_roller"] }
```

Comment:
I've forked log4rs into tari-project, it may be better to use that one

Reply:
Updated